airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/JdbcInsertFlushFunction.kt

```diff
@@ -15,9 +15,9 @@ class JdbcInsertFlushFunction(
     override val optimalBatchSizeBytes: Long
 ) : DestinationFlushFunction {
     @Throws(Exception::class)
-    override fun flush(desc: StreamDescriptor, stream: Stream<PartialAirbyteMessage>) {
+    override fun flush(decs: StreamDescriptor, stream: Stream<PartialAirbyteMessage>) {
```

Comment:
is this to match super declaration? I'd prefer to modify the base class to use `desc`, since presumably this is shorthand for "descriptor"